zend_abstract_interface/zai_string/string.h

```diff
@@ -11,13 +11,13 @@ typedef struct zai_string_view_s {
 } zai_string_view;
 
 #define ZAI_STRL_VIEW(cstr) \
-    (zai_string_view) { .len = sizeof(cstr) - 1, .ptr = (cstr) }
```

Comment:
Microsoft's C support has always been bad, but this is super annoying >.<

Reply:
Yes, it is. This branch is also still based on quite an old state of the repo, so I'll have to rebase once agent sampling is merged, which will include your newest zai changes...

Reply:
Fwiw, it's getting better in current (vs 2022), but we need to build older VS versions for PHP.

Reply:
Honestly, if we required a newer toolchain, which versions of PHP would be lost?

Reply:
All, except the upcoming PHP 8.3 :-)

Reply:
Hmm... I mean, that _is_ tempting. "Yeah, we support Windows now, but you gotta upgrade to PHP 8.3" ;)

But there may be a balance. According to this wiki entry, we could maybe use VS 2019 for PHP 8.X, and its C support does look a lot better:
https://learn.microsoft.com/en-us/cpp/overview/visual-cpp-language-conformance?view=msvc-160#c-standard-library-features-1